src/controllers/listItemController.js

```diff
@@ -0,0 +1,49 @@
+const dataService = require('../services/dataService/listItemFunctions')
+
+function findListItem(req, res, next) {
+  dataService.findListItem()
+    .then(listsItem => {
+      res.status(200).json(listsItem)
+    })
+    .catch(next)
+}
+
+function createListItem(req, res, next) {
+  dataService.createListItem()
+    .then(listsItem => {
+      res.status(200).json(listsItem)
+    })
+    .catch(next)
+}
+
+function deleteListItem(req, res, next) {
+  dataService.deleteListItem()
+    .then(listsItem => {
+      res.status(204).send("Successfully deleted")
+    })
+    .catch(next)
+}
+
+function updateListItem(req, res, next) {
+  dataService.updateListItem()
+    .then(listsItem => {
+      res.status(200).json(listsItem)
+    })
+    .catch(next)
+}
+
+function getListItem(req, res, next) {
+  dataService.getListItem()
```

Comment:
Again...should be passing in the item ID here. Can be extracted from the request.